invenio_rdm_records/config.py

```diff
@@ -308,3 +308,29 @@ def _(x):
 See :class:`invenio_rdm_records.metadata_extensions.MetadataExtensions` for
 allowed types and how this configuration variable is used.
 """
+
+RDM_RECORDS_CUSTOM_VOCABULARIES = {}
+"""Paths to custom controlled vocabularies.
+
+Of the shape:
+
+.. code-block:: python
+
+    {
+        '<vocabulary1>': '<absolute path to CSV file containing it',
```

Comment:
Question: Does it have to be absolute? Could it work with relative? Im just thinking if we will have to edit the Dockerfile, hardcode the path or it would just work specifying a relative path.

Reply:
I was wondering about this too. Especially given that the file might be in these different locations (linked and so on)... 

I am doing a full test today and that might be enlightening. If you or others have recommendations for locations / ways to refer to the file, please share.

Reply:
We would need an easy way to provide new CSV files in the instance's folder (i.e. it should be possible to provide them and commit them to inveniosoftware/demo-inveniordm

Reply:
What about a "vocabularies" folder? same way we have templates and so on?

Reply:
Would work yes.

Reply:
See what I did there: https://github.com/inveniosoftware/cookiecutter-invenio-rdm/pull/59 . Is that reasonable? This way we don't have to go back to modifying the cookiecutter and the cli everytime we want to add application data. `vocabularies/` would just be a folder in `app_data/`. 

I couldn't use `data/` because the name is already used for the local files storage folder...